packages/apps/reputation-oracle/server/src/modules/escrow-completion/payouts-calculation/cvat-payouts-calculator.ts

```diff
@@ -46,7 +46,15 @@ export class CvatPayoutsCalculator implements EscrowPayoutsCalculator {
       throw new Error('Invalid annotation meta');
     }
 
-    const jobBountyValue = ethers.parseUnits(manifest.job_bounty, 18);
+    const tokenAddress = await escrowClient.getTokenAddress(escrowAddress);
+    const tokenDecimals = await this.web3Service.getTokenDecimals(
+      chainId,
+      tokenAddress,
+    );
+    const jobBountyValue = ethers.parseUnits(
```

Comment:
Can you please add as a precaution make sure that `manifest.job_bounty` value matches the token decimals?
Otherwise, `.parseUnits()` will throw an error in case `job_bounty` decimals > token decimals.